tasks/copy_from_archive/copy_from_archive.py

```diff
@@ -8,13 +8,15 @@
 import time
 from typing import Any, Dict, List, Optional, Union
 
-# noinspection PyPackageRequirements
 import boto3
 import fastjsonschema
+
+# noinspection PyPackageRequirements
```

Comment:
This is used to mark requirements that will not appear in requirements.txt, such as boto3

Reply:
updated